google-cloud-speech/lib/google/cloud/speech/v1/speech_client.rb

```diff
@@ -138,9 +125,11 @@ def initialize \
               updater_proc = credentials.updater_proc
             end
 
+            package_version = Gem.loaded_specs['google-cloud-speech'].version.version
```

Comment:
Why not use `Google::Cloud::Speech::VERSION` here?

Reply:
GAPIC doesn't generate this constant and isn't aware of manual code.

Agree that it's a cleaner solution for GAPIC to generate the constant (although we would have to be careful not to overwrite it accidentally on refreshes). Filed https://github.com/googleapis/toolkit/issues/1749.

Reply:
This value is the same as `lib_version` being passed into the constructor, right? Why not just use that value here as well?

Or should the `gapic` version be signifying something else?

Reply:
`lib_name/lib_version` exist for historical reasons that don't apply to `google-cloud-ruby`. They actually will not be set for GAPIC-only APIs.